dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1185,10 +1188,9 @@ restoreWorkarea()
   fi
   rm -rf "${SERVER_OUTPUT_DIR}/workarea"
   # copy backup to workarea
-  cp -r --preserve=timestamps "${SERVER_OUTPUT_DIR}/workarea-checkpoint" "${SERVER_OUTPUT_DIR}/workarea"
+  cp -r --preserve=timestamps "${SERVER_OUTPUT_DIR}/checkpoint/workarea-checkpoint" "${SERVER_OUTPUT_DIR}/workarea"
   rc=$?
   # mv checkpoint directory back
-  mkdirs "${SERVER_OUTPUT_DIR}/workarea/"
```

Comment:
I had this line here incase the `cp` above failed and the `workarea` dir didn't get created at all during the `cp` command.  I was trying hard to make sure the `checkpoint` directory got moved back under workarea even if the `cp` failed. so we didn't leave junk under the output directory.

Reply:
Good point. I'll put back the line
```
mkdirs "${SERVER_OUTPUT_DIR}/workarea/"
```